chsdi/models/vector/__init__.py

```diff
@@ -52,6 +52,7 @@ class HTTPBandwidtLimited(exc.HTTPServerError):
 
 
 class Vector(GeoInterface):
+    __queryable_attributes__ = []
```

Comment:
Because of the following use case:

The attribute discovery service is not scale dependent (for query tool). 
The default behavior is to return all the attributes as defined on the model if `__queryable_attributes__` is not defined on the model.
Here we have a scale dependent layer with different attributes on the model according to the scale.
As we may use the attribute discovery service in other contexts than the query tool I would remove it from the parent class Vector and add it only on the classes where it's needed.